### PR TITLE
Cherry-pick Updated XR SDK documentation (#9173)

### PR DESCRIPTION
**Pull request by RogPodge**
* Updated to consistently refer to XR SDK, rewrote the getting started documentation

* fixed doc errors

* Update Documentation/GettingStartedWithMRTKAndXRSDK.md

Co-authored-by: Kurtis <kurtie@microsoft.com>

Co-authored-by: Kurtis <kurtie@microsoft.com>

## Changes
- Fixes: # .

### CODE REVIEW

File: Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs [hunk 1]
```diff
@@ -17,7 +17,9 @@ namespace Microsoft.MixedReality.Toolkit.XRSDK.Oculus.Input
     [MixedRealityDataProvider(
         typeof(IMixedRealityInputSystem),
         SupportedPlatforms.WindowsStandalone | SupportedPlatforms.Android,
-        "XRSDK Oculus Device Manager")]
+        "XR SDK Oculus Device Manager",
+        "Oculus/XRSDK/Profiles/DefaultOculusXRSDKDeviceManagerProfile.asset",
```
Comment (keveleigh) [inline Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs:21]: Looks like this has a different name in 2.5.3: https://github.com/microsoft/MixedRealityToolkit-Unity/tree/releases/2.5.3/Assets/MRTK/Providers/Oculus/XRSDK/Profiles

```suggestion
        "Oculus/XRSDK/Profiles/OculusXRSDKDeviceManagerProfile.asset",
```

Comment (RogPodge) [reply Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs:21]: Seems that this is needed based on this PR: https://github.com/microsoft/MixedRealityToolkit-Unity/commit/216ef024fd3405208cf41d05b594edb47734571b

Comment (RogPodge) [reply Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs:21]: I see that the asset is still named in the old way in 2.5.3 so you are right it should be OculusXRSDKDeviceManagerProfile.

**Review by keveleigh — APPROVED** [review]

**Pull request merged**